.travis.yml

```diff
@@ -180,4 +180,4 @@ after_success:
     - codecov
     # Prepare.release
     - doc/release/contribs.py HEAD~10
-    - bash tools/travis/deploy_docs.sh
+    # - bash tools/travis/deploy_docs.sh
```

Comment:
Please remove this line and `git rm` the `deploy_docs.sh` file which is not used elsewhere.